cmd/maya-apiserver/spc-watcher/start.go

```diff
@@ -37,6 +37,7 @@ var (
 type QueueLoad struct {
 	Key       string
 	Operation string
+	Object interface{}
```

Comment:
can we make use of storagepoolclaim type rather than the generic interface{} type?

Reply:
Lets keep it generic, and then type caste accordingly..
Any specific reason for not keeping the interface.

Reply:
@AmitKumarDas